client/src/views/users/ActiveUsersPage.vue

```diff
@@ -247,25 +249,7 @@ function getSelectedUsers(): UserInfo[] {
 }
 
 async function inviteUser(): Promise<void> {
-  const modal = await modalController.create({
-    component: CreateUserInvitationModal,
-    cssClass: 'create-user-invitation-modal',
-  });
-  modal.present();
-
-  const { data, role } = await modal.onWillDismiss();
-
-  if (role === 'confirm') {
-    const result = await parsecInviteUser(data);
-
-    if (result.ok) {
-      console.log('Invite successful', result.value);
-      // Redirect to invitations page
-      routerNavigateTo('invitations');
-    } else {
-      console.log(`Failed to invite ${data}`, result.error);
-    }
-  }
+  routerNavigateTo('invitations', {}, {openInvite: true});
```

Comment:
why remove the modal?

Reply:
Modal is handled by the `InvitationsPage` instead to avoid dupplicating the same invitation code with the same error handling. Instead, when the user clicks on the `Invite user` button in the `ActiveUserPage`, we redirect them to the `InvitationsPage` with a specific option (`optionInvite: true`) to tell the `InvitationsPage` to open the dialog.